packages/dependency-extraction-webpack-plugin/lib/util.js

```diff
@@ -64,6 +72,15 @@ function defaultRequestToHandle( request ) {
 
 		case 'lodash-es':
 			return 'lodash';
+
+		// Ignore object-assign ponyfill, which is used by jsx-(dev)-runtime.
```

Comment:
```suggestion
		// Ignore object-assign polyfill, which is used by jsx-(dev)-runtime.
```

https://github.com/sindresorhus/ponyfill seems to be a thing 😃

Reply:
[`object-assign`](https://github.com/sindresorhus/object-assign) IS a ponyfill though 😅 .

Reply:
> While polyfills are naughty, ponyfills are pure, just like ponies.